libr/core/cmd_search.c

```diff
@@ -736,6 +744,30 @@ static void print_rop (RCore *core, RList *hitlist, char mode, int *json_first)
 	if (mode != 'j') r_cons_newline ();
 }
 
+
+static int has_branch_delay_slot(const char* arch) {
+	
+	if (strncmp(arch,"mips",4)) 
```

Comment:
strncmp is done wrong, it returns 0 on equality

also, can you collapse this up a bit? not as many empty lines